ha/k8s_setup/ha_setup.py

```diff
@@ -186,7 +186,8 @@ def process(self):
                          'data_pod_label' : data_pod_label,
                          'MONITOR' : {'message_type' : 'k8s_event', 'producer_id' : 'k8s_monitor'},
                          'EVENT_MANAGER' : {'message_type' : 'health_events', 'producer_id' : 'system_health',
-                                            'consumer_group' : 'health_monitor', 'consumer_id' : '1'}
+                                            'consumer_group' : 'health_monitor', 'consumer_id' : '1'},
+                         'K8S:POD': {'namespace': 'cortx', 'resource_type': 'k8s:pod'}
```

Comment:
This resource type  will be hardcoded here and not configurable currently  right ? If tomorrow it is made configurable, we will get the type as node and not pod (since pod is not used in cortx env)  . We dont need to do anything right now, just mentioning this point here. 

Also the